cpu/nrf51822/include/nrfmin.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        drivers_nrf51822_nrfmin NRF Minimal Radio Driver
+ * @ingroup         drivers
+ * @brief           Minimal driver for the NRF51822 radio
+ * @{
+ *
+ * @file
+ * @brief           Interface definition for the nrfmin NRF51822 radio driver
+ *
+ * This driver enables the use of the NRF51822 radio in a 802.15.4 like fashion.
+ * The driver does however not support the 802.15.4 frame format, but uses its
+ * own packet format.
+ *
+ * The driver supports:
+ *   - short address (16-bit)
+ *   - using CPU-ID for default address
+ *   - address broadcast (broadcast address is ff:ff)
+ *   - PAN IDs (0 to 0xffff)
+ *   - setting of channel (0 to 0x3f)
+ *   - setting of TX power (+4dBm to -20dBm)
+ *   - packet type labeling
+ *   - setting device state (RX, SLEEP)
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NRFMIN_H_
+#define NRFMIN_H_
+
+#include "net/ng_netdev.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Default PAN ID used after initialization
+ */
+#define NRFMIN_DEFAULT_PAN          (0x0550)
+
+/**
+ * @brief   Default channel set after initialization
+ */
+#define NRFMIN_DEFAULT_CHANNEL      (1U)                /* 2401MHz */

```

Comment:
@haukepetersen Please look into the radio characteristics nRF51822_PS_v3.1.pdf

Chapter: 8.5.1 +/-320KHz typically frequency deviation at 2Mbps without min or max values, With BLE maximal +/-275KHz

Chapter 8.5.5 RX selectivity - modulated interfering signal
@2Mbps: 12db at channel, -4db at channel+-2MHz, -24db at channel+-4MHz, -28db at channel+-6MHz, -44db at channel+-12MHz
@BLE: 4db at channel, -10db at channel+-1MHz, -25db at channel+-2MHz 

I think both characteristics are reasons to use another channel. 2MBs uses a lot more bandwith then BLE modulation. Maybe it's safe to use this channel @250kbps but i don't know it. I'm sure there are to much submissions outside the ISM range with this configuration.